pkg/arvo/sys/hoon.hoon

```diff
@@ -1905,17 +1905,17 @@
 ++  map
   |$  [key value]                                       ::  table
   $|  (tree (pair key value))
-  |=(a=(tree (pair)) ~(apt by a))
+  |=(a=(tree (pair)) ?:(=(~ a) & ~(apt by a)))
```

Comment:
idk how this fixes the jet printfs, but seems a little bit wrong to fix those on the hoon side...

Reply:
This is a work around for the lack of hint pass-thru in the seminoun interpreter `|musk`. Calls to %fast-hinted functions from inside molds trigger the "parent not registered jet hint". And `|musk` would probably need to be changed significantly in order to pass %fast hints through.